build_runner/lib/src/server/hot_reload_client/client.dart

```diff
@@ -149,27 +148,29 @@ List<K> keys<K, V>(JsMap<K, V> map) {
   return List.from(_jsArrayFrom(map.keys()));
 }
 
-Future<Module> Function(String) _futurifyLoaderFunction(
-        void Function(String, void Function(Object), void Function(JsError))
-            loaderFunction) =>
-    (moduleId) {
-      var completer = Completer<Module>();
-      var stackTrace = StackTrace.current;
-      loaderFunction(moduleId, allowInterop((Object moduleObj) {
-        var moduleKeys = List<String>.from(_jsObjectKeys(moduleObj));
-        var moduleValues =
-            List<HotReloadableLibrary>.from(_jsObjectValues(moduleObj));
-        var moduleLibraries = moduleValues.map((x) => LibraryWrapper(x));
-        var module = Module(Map.fromIterables(moduleKeys, moduleLibraries));
-        completer.complete(module);
-      }),
-          allowInterop((e) => completer.completeError(
-              HotReloadFailedException(e.message), stackTrace)));
-      return completer.future;
-    };
-
-var _reloadModule = _futurifyLoaderFunction(dartLoader.forceLoadModule);
-var _loadModule = _futurifyLoaderFunction(dartLoader.loadModule);
+Module _getModuleLibraries(String moduleId) {
+  var moduleObj = dartLoader.getModuleLibraries(moduleId);
+  if (moduleObj == null)
```

Comment:
when the entire conditional doesn't fit on one line add the `{}`

https://www.dartlang.org/guides/language/effective-dart/style#do-use-curly-braces-for-all-flow-control-structures